six/modules/c++/scene/source/ProjectionModel.cpp

```diff
@@ -62,11 +62,19 @@ ProjectionModel(const Vector3& slantPlaneNormal,
     mSlantPlaneNormal(slantPlaneNormal),
     mSCP(scp),
     mARPPoly(arpPoly),
-    mARPVelPoly(mARPPoly.derivative()),
     mTimeCOAPoly(timeCOAPoly),
     mLookDir(lookDir),
     mErrors(errors)
 {
+    try
```

Comment:
Yes, it threw when the poly was empty, which took a bit of time to track down.  I've made the change.